blivet/devicefactory.py

```diff
@@ -1244,7 +1248,11 @@ def _get_device_size(self):
         return size
 
     def _set_device_size(self):
+        """ Set the size of the factory device. """
         size = self._get_device_size()
+
+        # self.raw_device = self.device.raw_device
```

Comment:
maybe use `==` so it's obvious this is a statement rather than a commented-out assignment